advanced_topics.qmd

```diff
@@ -330,42 +336,42 @@ And we can visualize the underlying data along with weighted mean and prediction
 plot <- ggplot(model_data) +
   aes(
     x = factor(
-      edu, 
```

Comment:
Ooph apparently I don't (yet) have positron on my desktop setup for doing this on save :/ I should do that